### PR TITLE
Multiplies opacity by [value] by default

### PR DESCRIPTION
**Pull request by fdansv**
Ref #2125 

By default, appends ```*[value]``` to the ```marker-fill-opacity``` rule of the torque visualisation's cartocss. This makes heatmaps be weighed by the value generated through ```-torque-aggregation-function```.

The change is not applied retroactively, but any change to the visualisation through the wizard will trigger it.

@javisantana 
cc @iriberri, @andrewxhill

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2617/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2618/))

File: lib/assets/javascripts/cartodb/table/menu_modules/carto_wizard_forms.js [hunk 1]
```diff
@@ -687,7 +687,7 @@ cdb.admin.forms.torque_heat = {
     {
       name: 'Opacity',
       form: {
-        'marker-opacity': { type: 'opacity' , value: 0.4 }
+        'marker-opacity': { type: 'opacity' , value: "0.4" }
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb/table/menu_modules/carto_wizard_forms.js]: oops, do the cast in the torque.js instead of changing the type here

**Comment by javisantana** [comment]
:es:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2619/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2620/))

**Pull request merged**